bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/admin/StoreAdminHandler.java

```diff
@@ -430,4 +433,46 @@ public Response delete( final @PathParam( "packageType" ) String packageType,
         return response;
     }
 
+    @ApiOperation( "Retrieve the definition of a remote by specific url" )
+    @ApiResponses( { @ApiResponse( code = 200, response = ArtifactStore.class, message = "The store definition" ),
+                           @ApiResponse( code = 404, message = "The remote repository doesn't exist" ), } )
+    @Path( "/query/byUrl" )
+    @GET
+    public Response getRemoteByUrl( final @PathParam( "packageType" ) String packageType,
+                                    final @ApiParam( allowableValues = "remote", required = true )
```

Comment:
I don't understand the meaning of this param. If there is only one allowed value and it really only makes sense for remote repos, I would remove this one.